lib/tasks/visitor_count.rake

```diff
@@ -0,0 +1,31 @@
+namespace :visitor_count do
+  desc "rails 'visitor_count:recalculate[from{DD/MM/YYYY}, duration{in minute}]'"
+  task :recalculate, [:from, :duration] => :environment do |task, args|
+    puts("Start date is required") || exit if args[:from].blank?
+    puts("Duration is required") || exit if args[:duration].blank? || args[:duration].to_i > 0
+
+    visited = Ahoy::Visit.from(args[:from].to_date, [])
```

Comment:
@radinreth as the 2nd parameter is optional so better call `visited = Ahoy::Visit.from(args[:from].to_date)` without passing the 2nd argument to make it more readable then passed []